status/status.go

```diff
@@ -292,15 +292,16 @@ func (s *Status) generateTestReport(ctx context.Context, detail intgteststat.Int
 	}
 
 	report := TestReport{
-		Text:           text,
-		FullName:       fullName,
-		ScenarioName:   detail.ScenarioName,
-		SnapshotName:   snapshot.Name,
-		ComponentName:  snapshot.Labels[gitops.SnapshotComponentLabel],
-		Status:         detail.Status,
-		Summary:        summary,
-		StartTime:      detail.StartTime,
-		CompletionTime: detail.CompletionTime,
+		Text:                text,
+		FullName:            fullName,
+		ScenarioName:        detail.ScenarioName,
+		SnapshotName:        snapshot.Name,
+		ComponentName:       snapshot.Labels[gitops.SnapshotComponentLabel],
+		Status:              detail.Status,
+		Summary:             summary,
+		StartTime:           detail.StartTime,
+		CompletionTime:      detail.CompletionTime,
+		TestPipelineRunName: detail.TestPipelineRunName,
```

Comment:
It looks like you added this in so that you could call `FormatPipelineURL` in `reporter_github.go`.  Why not just add a URL parameter to the report and call `FormatPipelineURL` here?

I ask because  this is the approach I'm taking in STONEINTG-842.  It works, but whatever we do should be the same so I can switch to your approach if there's some advantage to it.